index/miner/lotusidx/store/store.go

```diff
@@ -0,0 +1,189 @@
+package store
```

Comment:
So here's the new `store` that the Miner Index uses. Before, there wasn't a separate `store` package because the persistence was made in the same package as the miner index. So I took advantage and moved the persistence logic to this own package (as we usually do for better organization).